TwitchLib.Api.Helix/Tags.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using TwitchLib.Api.Core;
+using TwitchLib.Api.Core.Enums;
+using TwitchLib.Api.Core.Interfaces;
+using TwitchLib.Api.Helix.Models.Tags;
+
+namespace TwitchLib.Api.Helix
+{
+    public class Tags : ApiBase
+    {
+        public Tags(IApiSettings settings, IRateLimiter rateLimiter, IHttpCallHandler http) : base(settings, rateLimiter, http)
+        {
+        }
+
+        public Task<GetAllStreamTagsResponse> GetAllStreamTagsAsync(string after = null, int first = 20, List<string> tagIds = null, string accessToken = null)
+        {
+            var getParams = new List<KeyValuePair<string, string>>();
+            if(after != null) {
```

Comment:
Might want to format the document so it stays consistent --> 1 space between if and the opening bracket, curly bracket on next line.

Reply:
done